assets/test_gsMetZobe1.yaml

```diff
@@ -11,6 +11,11 @@ dataset:
   HiC:
     reads:
       - reads: /lustre/scratch123/tol/resources/genomeassembly/testdata/gsMetZobe1/hic-arima2/35528_4#7.cram
+  trio:
+    matreads:
+      - https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz
+    patreads:
+      - https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz
```

Comment:
As discussed, there is a separate trio test implemented. The 'trio:' section has to be removed here.

Reply:
ok, thanks both @ksenia-krasheninnikova @prototaxites  I will incorperate all your comments.